releases/kovan/1.2.10/index.html

```diff
@@ -0,0 +1,20 @@
+<html>
+	<head>
+		<title>Kovan Release 1.2.10</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Kovan Release 1.2.10</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/kovan/1.2.10/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/Qmc3VkuiwWNyPsdR3xzmwSs5aM24ff2jwGbjHPMS6Yt7dh">Contract addresses (IPFS hosted)</a></li>
```

Comment:
@brianmcmichael This doesn't look similar to Kovan contract addresses. For example, ETH-C is missing

Reply:
Old pin got moved up. Fixed.